### PR TITLE
Initialize CsrfToken for Yii apps

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | #2

### CODE REVIEW

File: config/events-web.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+use Psr\Container\ContainerInterface;
+use Yiisoft\Csrf\CsrfToken;
+
+return [
+    \Yiisoft\Yii\Web\Event\BeforeRequest::class => [
+        fn (object $event, ContainerInterface $container) => CsrfToken::initialize(),
```
Comment (viktorprogger) [inline config/events-web.php:10]:
```suggestion
        fn (\Yiisoft\Yii\Web\Event\BeforeRequest::class $event, ContainerInterface $container) => CsrfToken::initialize(),
```

**Comment by viktorprogger** [comment]
It could be good to document this right away

File: config/events-web.php (hunk 1 above)
Comment (yiiliveext) [reply config/events-web.php]: You don't use `ContainerInterface $container` in this case, remove it

**Comment by vjik** [comment]
> It could be good to document this right away

Add description for event.

File: config/events-web.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+declare(strict_types=1);
+
+use Yiisoft\Csrf\CsrfToken;
+use Yiisoft\Yii\Web\Event\BeforeRequest;
+
+return [
+    BeforeRequest::class => [
+        // Initialize static class CsrfToken for reset token when use an alternative way of running
+        // an application. For example, RoadRunner or Swoole.
```
Comment (viktorprogger) [inline config/events-web.php:11]:
```suggestion
        // Resets the token. It is relevant when requests are handled in a loop.
        // I.e. with RoadRunner, Swoole, Codeception.
```

**Review by viktorprogger — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Pull request merged**